src/Exception/ApiException.php

```diff
@@ -50,7 +50,7 @@ public function __construct(GuzzleRequestException $previous, $message = '')
 
         if ($this->previous->hasResponse()) {
             $this->response = $this->previous->getResponse();
-            $this->requestId = $this->response->getHeader('X-Contentful-Request-Id')[0];
+            $this->requestId = $this->response->getHeaderLine('X-Contentful-Request-Id');
```

Comment:
My concern were more around what happens if we have a bug and duplicate a header. (has happened before)

Of course with the current code we have the opposite problem that if we have a bug that the header isn't set if tails 🤔